tests/evaluator-tests.js

```diff
@@ -168,8 +168,8 @@ describe('Evaluator', () => {
       var currentProjectGetter = [
         ['projects'],
         ['session', 'currentProjectId'],
-        (projects, currentProjectId) => {
```

Comment:
Im not really a fan of this rule, we dont write a lot of large JS files with nested closures and sometimes using the same variable name is more correct than a "new" one.